src/RegExLib.Web/Endpoints/Expressions/Create.CreateExpressionCommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+using RegExLib.Web.ApiModels;
+
+namespace RegExLib.Web.Endpoints.Expressions
+{
+  public class CreateExpressionCommand
+  {
+    [Required]
+    public ExpressionDTO ExpressionDto { get; set; } = new ExpressionDTO();
```

Comment:
If you go with my suggestion above, this would be replaced with the DTO properties.

Reply:
Done in commit 36854